src/APIFunctions/3DPrinting.js

```diff
@@ -50,14 +60,14 @@ export async function submit3DPrintRequest (printRequest) {
  * @returns {(Object[]|string)} - The list of requests or error from the API.
```

Comment:
Be sure to update the documentation of this comment, we don't return a boolean anymore

```suggestion
 * @returns {ApiResponse} Containing any error information related to the 
 * request or list of requests
```